README.md

```diff
@@ -503,15 +503,44 @@ Output:
 
 #### FTP
 
-TKTKTK
+FTP operations are handled in the FTP container, using Ruby's `Net::FTP` module.
+
+The `URL` property is required. It should be formatted as follows:
+```
+ftp://user:password@host:port/path/file.extension
+```
+If port is not specified, it will default to the standard FTP command port, `21`.
+
+There are additional parameters to specify how the file is transferred:
+
+`MD5`: The default is `false`, to indicate the md5 file should not be written.
+If set to `true` Porter will write an md5 file, containing the md5 hash of the input file (e.g. `<input file>.md5`).
+This is useful both as a semaphore file, as it is written after the primary file is written successfully,
+and useful to validate the file was transferred without error by checking the md5 signature.
+
+`Passive`: the default mode is passive, but if this option is set to `false`, it will use active mode.
+The system also includes retry logic, and will try active mode if passive mode fails several times.
 
 Input:
 
 ```json
 {
     "Type": "Copy",
     "Mode": "FTP",
-    "URL": "ftp://foo:bar@example.com"
+    "URL": "ftp://foo:bar@example.com",
```

Comment:
Maybe have this match the example from the copy a little closer. Mainly having a file path in there, to make it clear that the URL isn't just for the server info. I think the FTP script always expects a filename.